tests/FacebookRequestTest.php

```diff
@@ -125,14 +125,14 @@ public function testAProperUrlWillBeGenerated()
 
         $getUrl = $getRequest->getUrl();
         $expectedParams = 'foo=bar&access_token=foo_token&appsecret_proof=df4256903ba4e23636cc142117aa632133d75c642bd2a68955be1443bd14deb9';
-        $expectedUrl = '/' . Facebook::DEFAULT_GRAPH_VERSION . '/foo?' . $expectedParams;
+        $expectedUrl = '/foo?' . $expectedParams;
 
         $this->assertEquals($expectedUrl, $getUrl);
 
         $postRequest = new FacebookRequest($app, 'foo_token', 'POST', '/bar', ['foo' => 'bar']);
 
         $postUrl = $postRequest->getUrl();
-        $expectedUrl = '/' . Facebook::DEFAULT_GRAPH_VERSION . '/bar';
+        $expectedUrl = '/bar';
 
         $this->assertEquals($expectedUrl, $postUrl);
```

Comment:
I'd add a case for a versionned request, as it useful in the case of batched request to use a different version than the one of the master request :)